commit 4d7a55f5b8099224e250e5e87882e4d7168af129
Author: Ming Lei <ming.lei@redhat.com>
Date:   Sat Sep 30 19:46:48 2017 +0800

    dm mpath: remove annoying message of 'blk_get_request() returned -11'
    
    commit 9dc112e2daf87b40607fd8d357e2d7de32290d45 upstream.
    
    It is very normal to see allocation failure, especially with blk-mq
    request_queues, so it's unnecessary to report this error and annoy
    people.
    
    In practice this 'blk_get_request() returned -11' error gets logged
    quite frequently when a blk-mq DM multipath device sees heavy IO.
    
    This change is marked for stable@ because the annoying message in
    question was included in stable@ commit 7083abbbf.
    
    Fixes: 7083abbbf ("dm mpath: avoid that path removal can trigger an infinite loop")
    Signed-off-by: Ming Lei <ming.lei@redhat.com>
    Signed-off-by: Mike Snitzer <snitzer@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/md/dm-mpath.c b/drivers/md/dm-mpath.c
index 11f273d2f018..e8094d8fbe0d 100644
--- a/drivers/md/dm-mpath.c
+++ b/drivers/md/dm-mpath.c
@@ -499,8 +499,6 @@ static int multipath_clone_and_map(struct dm_target *ti, struct request *rq,
 	if (IS_ERR(clone)) {
 		/* EBUSY, ENODEV or EWOULDBLOCK: requeue */
 		bool queue_dying = blk_queue_dying(q);
-		DMERR_LIMIT("blk_get_request() returned %ld%s - requeuing",
-			    PTR_ERR(clone), queue_dying ? " (path offline)" : "");
 		if (queue_dying) {
 			atomic_inc(&m->pg_init_in_progress);
 			activate_or_offline_path(pgpath);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1039_4d7a55f5b8099224e250e5e87882e4d7168af129.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1039_4d7a55f5b8099224e250e5e87882e4d7168af129.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
